modules/serverless-rn-1-25-0.adoc

```diff
@@ -36,3 +36,6 @@ It is recommended to not use the MT-Channel-Broker, but the Knative Kafka broker
 
 * The Federal Information Processing Standards (FIPS) mode is disabled for Kafka broker, Kafka source, and Kafka sink.
 * The `SinkBinding` object does not support custom revision names for services.
+* The Knative Serving Controller pod adds a new informer to watch secrets in the cluster. The informer includes the secrets in the cache, which increases memory consumption of the controller pod.
++
+If the pod runs out of memory, you can work around the issue by increasing the memory limit for the deployment.
```

Comment:
Do you think we can add the link [this section (Overriding Knative Serving system deployment configurations
)](https://docs.openshift.com/container-platform/4.11/serverless/admin_guide/serverless-configuration.html#knative-serving-CR-system-deployments_serverless-configuration) for how to increase the memory limit?
I thought that it is a little bit difficult to find the section so it will be kind if we add the link.

Reply:
@nak3 Unfortunately, we cannot add it directly in the known issue (docs guidelines). So it is added in the Additional resources section immediately after the note:
https://github.com/openshift/openshift-docs/pull/52588/files#diff-e15e61120290f3423707a9247d779df57f59ee4d4b0543ad38e73c80da6a69e1R31

Reply:
Oh, I understand! Thank you and sorry for my ignorance.